CONTRIBUTING.md

```
@@ -0,0 +1,561 @@
+open htmlcov/index.html
+```
+
+---
+
+## Submitting Changes
+
+### Commit Messages
+
+Use **Conventional Commits** format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)